docs/source/tools/export.rst

```diff
@@ -0,0 +1,164 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Ledger Export
+#############
+
+Export is currently an :doc:`Early Access Feature in Labs status </support/status-definitions>`.
+
+Introduction
+************
+
+Daml ledger exports read the transaction history or active contract set (ACS)
+from the ledger and write it to disk encoded as a
+:doc:`Daml Script </daml-script/index>` that will reproduce the ledger state
+when executed. This can be useful to migrate the history or state of a ledger
+from one network to another, or to replicate the ledger state locally for
+testing or debugging purposes.
+
+Usage
+*****
+
+The command to generate a Daml ledger export has the following form. ::
+
+  daml ledger export <format> <options>
+
+Right now Daml script, specified as ``script``, is the only supported export
+format. You can get an overview of the available command-line options using the
+``--help`` flag as follows. ::
+
+  daml ledger export script --help
+
+A full example invocation looks like this: ::
+
+  daml ledger export script --host localhost --port 6865 --party Alice --party Bob --output ../out --sdk-version 0.0.0
+
+The flags ``--host`` and ``--port`` define the Daml ledger to connect to. You
+can omit these flags if you are invoking the command from within a Daml project
+with a running ledger, e.g. with a running ``daml start``.
+
+The ``--party`` flags define which contracts will be included in the export. In
```

Comment:
definitely not for this PR but might be nice to accept comma-separated lists as well.